index.js

```diff
@@ -14,4 +16,8 @@ exports.compile = function (str, options) {
     Handlebars.registerHelper(helper, options.helpers[helper]);
   }
   return Handlebars.compile(str, options);
-}
\ No newline at end of file
+};
+exports.render = function _render(str, options, locals) {
```

Comment:
I'm not sure how you tested it but you need to `require('jstransformer')(require('./'))`. If that doesn't work I'll take a look soon (sorry I don't have my development machine rn).